lib/const.rb

```diff
@@ -346,8 +346,8 @@
   Chromebrew - Package manager for Chrome OS https://chromebrew.github.io
 
   Usage:
-    crew build [options] [-k|--keep] [-v|--verbose] <name> ...
-    crew check [-V|--version] [-v|--verbose] <name> ...
+    crew build [options] [-f|--force] [-k|--keep] [-v|--verbose] <name> ...
+    crew check [-f|--force] <name> ...
```

Comment:
Keep the verbose option?

Reply:
It never did anything, so removing it just makes things less misleading.